src/@sdk/queries/products.ts

```diff
@@ -55,7 +55,7 @@ export const productListDetails = gql`
       edges {
         node {
           ...BasicProductFields
-          price {
+          basePrice {
```

Comment:
The product's price is a range since we can override the price for one of its variants. To get this range we can use:

```
pricing {
  priceRange {
    start {
      net {
        amount
      }
    }
    stop {
      net {
        amount
      }
    }
  }
}
```

Example response:
```
"pricing": {
  "priceRange": {
    "start": {
      "net": {
        "amount": 1.8
      }
    },
    "stop": {
      "net": {
        "amount": 4.2
      }
    }
  }
}
```

In this case, in the product page, we should render the price as "From $1.8". If the amounts were equal then we could render the precise value. That's the logic we used to have in the old storefront in the good old Django-templates times ;)

Reply:
I fixed it with rendering range e.g. "1,80 USD - 4,20 USD" if values are not equal, similarly as it is rendered when variants exist.

Reply:
But I used gross value... Does it matter?

Reply:
It's fine to use gross prices for now. Eventually, it should depend on this setting:
```
{
  shop {
    displayGrossPrices
  }
}
```
But it can be done in a separate PR (we have an issue for that)